runsc/cmd/nvproxy/nvproxy.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package nvproxy provides subcommands for the nvproxy command.
+package nvproxy
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/google/subcommands"
+	"gvisor.dev/gvisor/pkg/sentry/seccheck"
+	"gvisor.dev/gvisor/runsc/flag"
+)
+
+type Nvproxy struct{}
+
+func (*Nvproxy) Name() string {
+	return "nvproxy"
+}
+
+func (*Nvproxy) Synopsis() string {
+	return "shows information about nvproxy support"
+}
+
+func (*Nvproxy) Usage() string {
+	buf := bytes.Buffer{}
+	buf.WriteString("Usage: nvproxy <flags> <subcommand> <subcommand args>\n\n")
+
+	cdr := createCommander(&flag.FlagSet{})
+	cdr.VisitGroups(func(grp *subcommands.CommandGroup) {
+		cdr.ExplainGroup(&buf, grp)
+	})
+
+	return buf.String()
+}
+
+func (*Nvproxy) SetFlags(*flag.FlagSet) {}
+
+func (*Nvproxy) Execute(ctx context.Context, f *flag.FlagSet, args ...any) subcommands.ExitStatus {
+	seccheck.Initialize()
```

Comment:
Why is seccheck needed? Seems like a copy from runsc/cmd/trace/trace.go. That command needs seccheck, but IDT it is needed here. We only want nvproxy. I think you should call nvproxy.Init() here (assuming all nvproxy commands need nvproxy package initialized) and remove it from listDrivers.Execute().

Reply:
It was indeed a copy-paste, I didn't check what it did 😅